### PR TITLE
Feature/#10 remaining time

### LINKED ISSUE #10: Feature/#10 Remaining time - rewrite this function from StarWars project using TS
Add time countdown to the game - one player's round

### PR DESCRIPTION
**Pull request by Enessetere**
closes #10

### CODE REVIEW

File: src/app/Timer.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export class Timer {
+  private maxTurnTime: number;
+  private turnTimeLeft: number;
+
+  constructor(turnTime = 15) {
+    this.maxTurnTime = turnTime;
+    this.turnTimeLeft = this.maxTurnTime;
+  }
```
Comment (synowa) [inline src/app/Timer.ts]: Consider those changes - it'll save two lines and one variable still doing the same work.
```.ts
// private maxTurnTime: number;
 private turnTimeLeft: number;

 constructor(private maxTurnTime = 15) {
 this.turnTimeLeft = this.maxTurnTime;
 }

**Review by synowa — APPROVED** [review]
Nicely done with testing time feature 🚀 
I added one suggestion at the beginning - consider changing that before a merge.

File: src/app/Timer.ts (hunk 1 above)
Comment (Enessetere) [reply src/app/Timer.ts]: good point, changed it already :)

**Review by DominikNowak — APPROVED** [review]
Looks good, well done :) For future refactor we need to think where to keep variables which will be changeable in settings like maxTurnTime.

**Pull request merged**